src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -74,6 +77,21 @@ function pipeDrupalPagesIntoMetalsmith(contentData, files) {
     };
   }
 
+  for (const navItem of taxonomies) {
+    const { entityBundle, name } = navItem;
+
+    if (name === 'Health Care') {
+      navItems[`${entityBundle}`] = navItem;
+    }
+
+    files['drupal/sidebar_navigation/index.html'] = {
```

Comment:
I think you can move this code creating the sidebar nav file outside of the for loop.